docs/index.md

```diff
@@ -223,17 +224,17 @@ $ ./node_modules/.bin/mocha mocha.test.js
 
 ## Assertions
 
-Mocha allows you to use any assertion library you wish.  In the above example, we're using Node.js' built-in [assert](https://nodejs.org/api/assert.html) module--but generally, if it throws an `Error`, it will work! This means you can use libraries such as:
+Mocha allows you to use any assertion library you wish. In the above example, we're using Node.js' built-in [assert](https://nodejs.org/api/assert.html) module--but generally, if it throws an `Error`, it will work! This means you can use libraries such as:
 
 - [should.js](https://github.com/shouldjs/should.js) - BDD style shown throughout these docs
```

Comment:
We're using "should.js" in the docs?

Reply:
I think this is a list of popular assertion libs?

Reply:
Maybe just misread the description "BDD style shown throughout these docs".